highfidelity/hifi
# Tablet home paginate

https://highfidelity.fogbugz.com/f/cases/7065/Paginate-Apps-on-Home-Screen

Test steps:
- Open Tablet home screen
- Add Tablet apps > 12 items
- Make sure page created and page indicator reflects add more pages
- Swipe pages right and left
- Make sure pages swiped correctly and page indicator reflects this
